filebeat/filebeat.yml

```diff
@@ -196,6 +199,20 @@ output:
     # [filebeat-]YYYY.MM.DD keys.
     #index: "filebeat"
 
+    # A template is used to set the mapping in elasticsearch
+    # By default template loading is disabled an not template is loaded.
+    # These settings can be adjusted to load your own template or overwrite existing ones
+    #template:
+
+      # Template name. By default the template name is the same as the filebeat
```

Comment:
Could you say simply, "By default the template name is Filebeat." Or you might say, "By default the template name is the same as the Beat name."